openedx/core/djangoapps/dark_lang/tests.py

```diff
@@ -242,7 +243,9 @@ def assert_cookie_lang_equals(self, value):
         """
         Assert that the language set in cookies is equal to value
         """
-        assert value == self.client.cookies.get(settings.LANGUAGE_COOKIE_NAME).value
+        lang_cookie = self.client.cookies.get(settings.LANGUAGE_COOKIE_NAME)
+        lang_cookie = lang_cookie.value if lang_cookie is not None and lang_cookie.value != '' else UNSET
```

Comment:
Renaming this variable to `lang_cookie_value` would make this more clear.

Reply:
Done